Generator/templates/declarations.template

```diff
@@ -23,7 +23,7 @@ extern const char NETWORK_INTERFACE[];
 
 #define ONE_MS 1000000		// used to simplify timer defn.
 #define Log_Debug(f_, ...) dx_Log_Debug((f_), ##__VA_ARGS__)
-static char Log_Debug_buffer[128];
+static char Log_Debug_buffer[256];
```

Comment:
I increased this buffer size because my IoTConnect telemetry debug was over 128 btyes and was truncated because the buffer was too small.